storage/innobase/include/buf0buf.h

```diff
@@ -1327,7 +1327,6 @@ class buf_page_t {
   unsigned access_time;
 
   bool is_corrupt;
-  bool encrypted; /*!< page is still encrypted */
```

Comment:
Good that this is removed. Was not a big fan of encryption attribute in buffer pool

Is the buf_page_t->encrypted not used anywhere else other than recovery?

Reply:
Yes, 'encrypted' member of buf_page_t was used only in recovery ('if' condition), but was never initialized (initialized by default to '0' by zmalloc)